Materials overhaul (#69)
bombasticSlacks/Aspirant: Game/Materials.md

## Material

> All things are made up of materials. This usually isn't a major concern to the average adventurer, but a savvy tradesman will want to carefully select materials of varying qualities for their projects. Characters will need to source materials for anything they are [Crafting](Crafting). Materials should be simple and interchangeable withing a quality.

Reviewer: Third last word, withing -> within

### Material Quality
All materials generally come in three quality levels:

#### Basic

> simple materials, not very useful for crafting anything but the most basic of things.

Reviewer: Missed capital "S"

#### Artisan
Well treated, high-quality materials used for proper tools and equipment.
#### Exotic
Impressive materials, sourced from unique locations and situations with powerful properties.

### Material Types
There are generally 8 types of materials that will be used for creating:

#### Bits

> After killing a creature you can, unless it’s been annihilated, usually [harvest](Equipment#Looting) some of what are called bits. Bits are a generalization of organs, glands, bones, and other valuables. Your character would intuitively know what parts of a creature are valuable, even if you don’t yet. 

Reviewer: Readability change:  "[...] usually harvest some of what are called bits." -> "[...] usually harvest some bits" || "[...] usually harvest bits"

##### Basic Bits
Simple unimpressive creature viscera, not useful for much more than cooking or filler.

> Blood, mundane organs, bones

##### Artisan Bits
Purposeful materials from creatures with clearly developed biological systems doing useful things.

> > An undead creature's icor, the claws of a powerful predator, eyes of a creature that can see in the dark

Reviewer: "ichor," probably spell-check related

##### Exotic Bits
Similarly highly developed biological systems, but doing truly incredible things.

> The regenerating heart of a golem, the blood of an elder vampire, the fire spewing lungs of a dragon.

---

#### Flora
The world is full of powerful and magical materials for characters to scavenge. Some even grow in the ground. Flora can be used for all sorts of medical and chemical processes.

##### Basic Flora
Simple plants useful for herbal remedies and basic purposes.

> Wild flowers, tobaco

##### Artisan Flora
Specialized plants with great properties possible to derive.

> > Psychadelic mushrooms, poppeys, powerful medicinal herbs

Reviewer: "poppey" -> "poppy"

##### Exotic Flora

> Plants with truley magical properties.

Reviewer: "truley" -> "truly"

> Glowbulb, firegrass, noxgreen

---

#### Metal
Rigid workable materials, making great [Armour](Armour), [Gear](Gear), and [Weapons](Weapons). Generally harvested from deposits in the ground or reclaimed.

##### Basic Metal
Poorly sourced metal that isn't very useful for working.

> Junk metal melted down, pig iron, poorly sourced metal.

##### Artisan Metals
Simple unimpressive metals that are built for purpose.

> Refined iron, copper, brass.

##### Exotic Metals

> Complex impressive metals with properties great for working with.

Reviewer: Readability suggestion:  "Complex impressive metals with properties great for working with." -> "Complex, impressive metals with great working properties."

Author: ```suggestion
Complex, impressive metals with great working properties.
```

> Steel, mithril, dwarven ingots.

---

#### Leather

> Repurposed skin of creature's, leather makes an excellent material for [Storage](Storage), and [Armour](Armour).

Reviewer: "creature's" -> "creatures"

Author: ```suggestion
Repurposed skin of creatures, leather makes an excellent material for [Storage](Storage), and [Armour](Armour).
```

##### Basic Leather
Simple animal leathers.

> > cow leather, deer hide.

Reviewer: Caps "Cow"

Author: ```suggestion
> Cow leather, deer hide.
```

##### Artisan Leather
Hardened leathers.

> > boiled leather, treated with epoxy, other treatments. 

Reviewer: Caps "Boiled"

Author: ```suggestion
> Boiled leather, treated with epoxy, other treatments. 
```

##### Exotic Leather
Exotic creature skins with stranger properties.

> > Dragon hide, demons skin.

Reviewer: "demons skin" -> "demon's skin"
Suggestion:  make both examples possessive or neither, i.e., "Dragon's hide, demon's hide" OR "Dragon hide, demon skin"

Author: ```suggestion
> Dragon hide, demon skin.
```

---

#### Wood
Sturdy natural resource generally reclaimed or harvested from trees, useful for the construction of some [Weapons](Weapons), and [Assets](Assets).

##### Basic Wood
High-quality wood that can be worked (anything worse than a hard wood isn't useful for crafting).

> Hard woods

##### Artisan Wood
High-quality wood that has been further treated with alchemical processes.

> Varnished, epoxied.

##### Exotic Wood
Magical wood, from ancient forests.

> Elven wood, spirit wood, living logs.

---

#### Resonant
Items with a clear magical attunement, pieces of magical items, mundane items collected from places of power and rare gemstones. Useful in powering [Focus](Example-Gear#Focus), [Ritual](Ritual), and advanced crafting.

##### Basic Resonant
Otherwise unusable pieces of damaged artifacts, fetishes, and other superstitious items.

> Shards of broken mirror, shavings of an ancient device

##### Artisan Resonant
Gemstones and fancy powders.

> Ruby, Sapphire, Diamond shards / stones

##### Exotic Resonant
Intentionally imbued powders, items warped from long exposure to enchanted places.

> Crystallized magic, liquified elements

---

#### Liquid

> Spirits, elixirs, strange fluids, anything that can be used to dissolve material, treat it or as lubrication. Useful in crafting [Comestibles](Comestibles).

Reviewer: Readability suggestion: "Spirits, elixirs, strange fluids, or anything that can be used to dissolve, treat, or lubricate materials."

Author: ```suggestion
Spirits, elixirs, strange fluids, or anything that can be used to dissolve, treat, or lubricate materials. Useful in crafting [Comestibles](Comestibles).
```
